include/MySQL_HostGroups_Manager.h

```diff
@@ -312,6 +678,23 @@ class MySQL_HostGroups_Manager {
 	pthread_mutex_t mysql_errors_mutex;
 	umap_mysql_errors mysql_errors_umap;
 
+	/**
+	 * @brief Initalizes the prometheus counters specified in hg_metrics_map.
+	 */
+	void init_prometheus_counters();
+	/**
+	 * @brief Initalizes the prometheus gauges specified in hg_metrics_map.
+	 */
+	void init_prometheus_gauges();
```

Comment:
We have these 2 functions defined in 5 different modules.
Let's avoid unnecessary copy/paste, and let's make them global.

Reply:
Refactored!